### PR TITLE
Ps 5.7 myr merge january2017

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

**Comment by george-lorch** [comment]
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/665/

File: mysql-test/suite/rocksdb.sys_vars/t/rocksdb_max_background_compactions_basic.test [hunk 1]
```diff
@@ -1,7 +1,16 @@
 --source include/have_rocksdb_as_default.inc
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb.sys_vars/t/rocksdb_max_background_compactions_basic.test:1]: --source include/have_myisam.inc

File: mysql-test/suite/rocksdb.sys_vars/t/rocksdb_persistent_cache_path_basic.test [hunk 2]
```diff
@@ -0,0 +1,16 @@
+--source include/have_rocksdb_as_default.inc
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb.sys_vars/t/rocksdb_persistent_cache_path_basic.test:1]: --source include/have_myisam.inc

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
5.6 comments + couple of specific 5.7 comments, which we should merge as there is no 5.7 upstream

**Pull request merged**